core/repository/consensus/event_repository.cpp

```diff
@@ -4,39 +4,50 @@
 #include <string>
 #include "../../util/convert_string.hpp"
 #include "../../model/transactions/abstract_transaction.hpp"
+#include "../../consensus/consensus_event.hpp"
 
 namespace repository{
 namespace event {
 
-
-  bool add(const std::string& hash, std::unique_ptr<abstract_transaction::AbstractTransaction> tx){
-    if(tx->getType() == abstract_transaction::TransactionType::message){
-       /*
-        world_state_repository::add(
-            hash,
-            convert::to_string<
-              abstract_transaction::AbstractTransaction
-            >(std::move(tx))
-        );
-        */
-    }
+  bool add(const std::string& hash, std::unique_ptr<consensus_event::ConsensusEvent> tx){
+//    if(tx->getType() == consensus_event::ConsensusEvent->tx::message){
+//       /*
+//        world_state_repository::add(
+//            hash,
+//            convert::to_string<
+//              consensus_event::ConsensusEvent
+//            >(std::move(tx))
+//        );
+//        */
+//    }
+    return false;
   }
 
-  bool update(const std::string& hash, const abstract_transaction::AbstractTransaction& tx){
-      
+  bool update(const std::string& hash, const consensus_event::ConsensusEvent& consensusEvent){
+    return false;
   }
-  bool remove(const std::string& hash){
 
+  bool remove(const std::string& hash){
+    return false;
   }
-  bool empty(){
 
+  bool empty(){
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The function 'empty' is never used.](https://www.codacy.com/app/soramitsu/iroha/file/4256044953/issues/source?bid=3712501&fileBranchId=3831026#l34)